cpp/src/barretenberg/stdlib/primitives/biggroup/biggroup.hpp

```diff
@@ -252,20 +252,18 @@ template <class Composer, class Fq, class Fr, class NativeGroup> class element {
     Fq y;
 
   private:
-    template <size_t num_elements,
-              typename = typename std::enable_if<std::is_same<Composer, plonk::UltraComposer>::value>>
+    template <size_t num_elements, typename = typename std::enable_if<Composer::type == ComposerType::PLOOKUP>>
     static std::array<twin_rom_table<Composer>, 5> create_group_element_rom_tables(
         const std::array<element, num_elements>& elements, std::array<uint256_t, 8>& limb_max);
 
-    template <size_t num_elements,
-              typename = typename std::enable_if<std::is_same<Composer, plonk::UltraComposer>::value>>
+    template <size_t num_elements, typename = typename std::enable_if<Composer::type == ComposerType::PLOOKUP>>
     static element read_group_element_rom_tables(const std::array<twin_rom_table<Composer>, 5>& tables,
                                                  const field_t<Composer>& index,
                                                  const std::array<uint256_t, 8>& limb_max);
 
     static std::pair<element, element> compute_offset_generators(const size_t num_rounds);
 
-    template <typename = typename std::enable_if<std::is_same<Composer, plonk::UltraComposer>::value>>
+    template <typename = typename std::enable_if<Composer::type == ComposerType::PLOOKUP>>
```

Comment:
Just flagging this `ComposerType::PLOOKUP`. If this construct is sticking around should it at least be changed to `ULTRA`?

Reply:
We probably should rename it, yes